lib/Cleantalk/Common/Firewall/FirewallUpdater.php

```diff
@@ -165,13 +167,18 @@ public function update()
 
                     // Do writing to the DB
                     reset( $lines );
+                    $subnets_count_from_stats = isset($helper::getFwStats()['firewall_subnets_count'])
```

Comment:
The condition `isset($helper::getFwStats()['firewall_subnets_count'])` is not enough. Need to compare on 0 too: `$helper::getFwStats()['firewall_subnets_count'] > 0`

Reply:
Fixed